modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/test/java/com/liferay/dynamic/data/mapping/data/provider/internal/DDMDataProviderInvokerImplTest.java

```diff
@@ -487,4 +550,65 @@ public void testTimeOutException() {
 			ddmDataProviderResponse.getStatus());
 	}
 
+	protected void setUpLanguageUtil() {
+		LanguageUtil languageUtil = new LanguageUtil();
+
+		languageUtil.setLanguage(PowerMockito.mock(Language.class));
+	}
+
+	protected void setUpPortalUtil() {
+		PortalUtil portalUtil = new PortalUtil();
+
+		Portal portal = PowerMockito.mock(Portal.class);
+		ResourceBundle resourceBundle = PowerMockito.mock(ResourceBundle.class);
+
+		PowerMockito.when(
+			portal.getResourceBundle(Matchers.any(Locale.class))
+		).thenReturn(
+			resourceBundle
+		);
+
+		portalUtil.setPortal(portal);
+	}
+
+	protected void setUpResourceBundleUtil() {
+		PowerMockito.mockStatic(ResourceBundleUtil.class);
+
+		PowerMockito.when(
+			ResourceBundleUtil.getBundle(
+				Matchers.anyString(), Matchers.any(Locale.class),
+				Matchers.any(ClassLoader.class))
+		).thenReturn(
+			ResourceBundleUtil.EMPTY_RESOURCE_BUNDLE
+		);
+	}
+
+	private DDMRESTDataProviderSettings
+		_createDDMRESTDataProviderSettingsWithTimeout(int timeout) {
+
+		DDMForm ddmForm = DDMFormFactory.create(
+			DDMRESTDataProviderSettings.class);
+
+		DDMFormValues ddmFormValues = DDMFormValuesTestUtil.createDDMFormValues(
+			ddmForm);
+
+		ddmFormValues.addDDMFormFieldValue(
+			DDMFormValuesTestUtil.createUnlocalizedDDMFormFieldValue(
+				"timeout", String.valueOf(timeout)));
+
+		return DDMFormInstanceFactory.create(
+			DDMRESTDataProviderSettings.class, ddmFormValues);
+	}
+
+	private int _getExecutionTimeoutInMilliseconds(
+		DDMDataProviderInvokeCommand command1) {
```

Comment:
@cleydyr, can we rename the parameter `command1` to `ddmDataProviderInvokeCommand`? Usually, this it the pattern that Brian appreciates. :slightly_smiling_face:

Reply:
Oh, yeah. This is an automatic refactor leftover. Nice catch.

Reply:
Hey @cleydyr, I believe that this pull has big chances to be merged without changes if we rename this parameter to `ddmDataProviderInvokeCommand` instead of `command`. :slightly_smiling_face: If you prefer, I can forward without this renaming, but Brian can ask you to change. Please let me know your decision.

Thanks.

Reply:
Significa. I'll add a rename commit.

Reply:
Do you think we'll have to rename `properties` as well in this case as well for the same reason?

Reply:
Makes sense, @cleydyr, but once you're setting the value using a method named `getProperties` than it's ok. But your suggestion is valid too. :wink:

Reply:
@cleydyr, feel free to keep the variable name as `properties` or rename it to `hystrixCommandProperties`. :wink: